src/Storage/FileSystemStorage.php

```diff
@@ -41,11 +41,22 @@ protected function doRemove(PropertyMapping $mapping, ?string $dir, string $name
     {
         $file = $this->doResolvePath($mapping, $dir, $name);
 
-        return \file_exists($file) && \unlink($file);
+        if (!\file_exists($file)) {
+            throw new \Exception("File does not exist: ".$file);
+        }
+        if (!\unlink($file)) {
```

Comment:
We don't need separate exceptions here

Reply:
OK, my rationale for doing so is that it provides a better debugging experience. I'll merge the two.

Reply:
(I plan to merge the commits when you're happy btw)

Reply:
This is linked below, in case you missed it: https://github.styleci.io/analyses/DynDKg?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Thanks! I didn't see that link no. I just applied the patch.